provider_packages/refactor_provider_packages.py

```diff
@@ -435,6 +435,62 @@ def amazon_package_filter(node: LN, capture: Capture, filename: Filename) -> boo
             rename("airflow.providers.amazon.common.utils.email")
         )
 
+    def refactor_elasticsearch_package(self):
```

Comment:
Is this function used somewhere?

Reply:
Doh! Forgot to invoke it down below 🤦‍♂️